### DISCUSSION TITLE
How to remove empty columns

### DISCUSSION THREAD

**Discussion by ElemenT5ven** [vaexio/vaex#1540] [Q&A]

**Comment by ElemenT5ven** [comment]
Hello,
I didn't find in the documentation a optimale way to remove empty columns from a dataframe.
Do you a have a few tips ?

Regards,

**Comment by JovanVeljanoski** [comment]
Hi, 

There  is `df.drop(...)` that removes columns of your choice. 

As to how to find empty columns..  i guess something like:
```python
n_samples = len(df)
empty_cols = [col for col in df.column_names if df[col].countna() == n_samples]
```
Or something similar.

Also, for general questions such as this we want to use the Discussions, or StackOverflow.

**Comment by ElemenT5ven** [comment]
Oh, thank you sorry for this

**Reply by maartenbreddels** [reply]
No problem, it is now moved to discussions.
